src/App.jsx

```diff
@@ -191,8 +204,10 @@ function KanbanBoard() {
     const dueDate = new Date();
     dueDate.setDate(dueDate.getDate() + 7); // Default to 1 week from now
     
+    const taskId = String(crypto.randomUUID?.() || Date.now() + Math.random());
+    
     await addTask(columnId, {
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
const ensureStringId = (id) => String(id);
const generateId = () => String(crypto.randomUUID?.() || Date.now() + Math.random());

const taskId = generateId();

await moveTask(
  ensureStringId(source.droppableId),
  ensureStringId(destination.droppableId),
  ensureStringId(result.draggableId),
```
The String() conversion for IDs is scattered throughout the code. Consider centralizing ID type handling in a utility function to ensure consistent type conversion and validation.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;